src/Microsoft.DotNet.VersionTools.net45/Automation/GitHubApi/GitHubClient.Desktop.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net;
+
+namespace Microsoft.DotNet.VersionTools.Automation.GitHubApi
+{
+    public partial class GitHubClient : IGitHubClient, IDisposable
+    {
+        static GitHubClient()
+        {
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
```

Comment:
@davidsh When targeting 4.5(.0) is this the right answer, or can/should it be assigned to 0?

Reply:
If you're targeting .NET 4.5, then the .NET default for this property is "Ssl30 | Tls10".  If targeting newer .NET versions, then this property defaults to either "Tls10 | Tls11 | Tls12" or "SystemDefault".

If your goal is just to get TLS 1.2 being negotiated and you don't want to change the .NET Framework targeting to something newer, then yes this is good.  But it is not future-proof, i.e. when TLS 1.3 becomes the right thing, then this code would need to change.

Reply:
BuildTools is fragile enough that I think it's worth keeping this net45-specific change for now.